test/functional/test_framework/test_wrapper.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2019 The Bitcoin Core developers
```

Comment:
nit: (c) 2019 (unless you've been secretly working on this for 5 years)

Reply:
Almost feels like it ;)